framework/src/base/ResidualObject.C

```diff
@@ -38,13 +38,12 @@ ResidualObject::ResidualObject(const InputParameters & parameters, bool is_nodal
     // VPPs used by ScalarKernels must be broadcast because we don't know where the
     // ScalarKernel will end up being evaluated
     // Note: residual objects should have a valid _moose_base.
-    VectorPostprocessorInterface(this,
-                                 parameters.get<std::string>("_moose_base") == "ScalarKernel"),
+    VectorPostprocessorInterface(this, getBase() == "ScalarKernel"),
     RandomInterface(parameters,
                     *parameters.getCheckedPointerParam<FEProblemBase *>("_fe_problem_base"),
                     parameters.get<THREAD_ID>("_tid"),
                     is_nodal),
-    Restartable(this, parameters.get<std::string>("_moose_base") + "s"),
+    Restartable(this, getBase() + "s"),
```

Comment:
The `+ "s"` seems gratuitous. Not all plural nouns are formed by appending an _s_. This might not be an issue now, but apps might define systems that will result in wrong plural forms. Albeit, just a minor cosmetic issue.

Reply:
But... this wasn't me